apps/settings/style/icons.css

```diff
@@ -293,8 +293,9 @@ aside.connecting {
  */
 
 html[dir="rtl"] a.menu-item:after {
-  left: 0.3rem;
+  left: -0.9rem;
   right: unset;
+  content: "back-light";
```

Comment:
To match the current direction of transitions the change is not required. For RTL we should still use "forward-light".